tests/draft4/refRemote.json

```diff
@@ -60,15 +60,77 @@
         },
         "tests": [
             {
-                "description": "changed scope ref valid",
+                "description": "base URI change ref valid",
                 "data": [[1]],
                 "valid": true
             },
             {
-                "description": "changed scope ref invalid",
+                "description": "base URI change ref invalid",
                 "data": [["a"]],
                 "valid": false
             }
         ]
+    },
+    {
+        "description": "base URI change - change folder",
+        "schema": {
+            "id": "http://localhost:1234/scope_change_defs1.json",
+            "type" : "object",
+            "properties": {
+                "list": {"$ref": "#/definitions/baz"}
+            },
+            "definitions": {
+                "baz": {
+                    "id": "folder/",
+                    "type": "array",
+                    "items": {"$ref": "folderInteger.json"}
```

Comment:
@Relequestual there is a convention already used in other tests to assume that  schemas at `http://localhost:1234/` are files in `remotes` folder, `folderInteger.json` is in `remotes/folder`, its full URI is `http://localhost:1234/folder/folderInteger.json` (and there is another existing test that points to it).